src/converter.cpp

```diff
@@ -80,24 +84,28 @@ struct MVD2Infos{
 
     void parseNeuron(const char* line){
         assert(neuron_line < vec_xyzr.size());
-        int unused, mtype, etype;
-        std::string morpho_name, metype;
+        int unused, hypercolumn, minicolumn, layer, mtype, etype;
```

Comment:
As far as I know, MVD2 does not restrict these values to be non-negative, and -1 could be used for stub values occasionally.